setup.py

```diff
@@ -275,6 +275,14 @@ def get_dynamic_setup_params():
             'config/base.yml',
             'config/module_defaults.yml',
         ],
+        'ansible_test': [
+            '*/*.*',
+            '*/*/*.*',
+            '*/*/*/*.*',
+            '*/*/*/*/*.*',
+            '*/*/*/*/*/*.*',
+            '*/*/*/*/*/*/*.*'
```

Comment:
Files without extensions need to be included as well. For example, all the files in this directory:

https://github.com/ansible/ansible/tree/devel/test/lib/ansible_test/_data/sanity/pylint/config

Reply:
We'll also want an `exclude_package_data` entry to exclude all of `test/lib/ansible_test/tests/`.

Reply:
I think you probably want a function which does this rather than trying to hardcode wildcarded strings in here.

Reply:
handled this a different way: ditched package_data altogether and updated sanity test accordingly